lambdas/postgres-migration-async-operation/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
```

Comment:
Why is this coming over wholesale?   Are we re-adding removed content here?

Reply:
Yeah, same thing happened with the data-migration lambdas. Didn't catch it here. 

https://github.com/nasa/cumulus/pull/3225/commits/f6ec3a42afde116a290c05556fed107bb0c2a952